Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -73,33 +73,30 @@ namespace OpenInfraPlatform {
 				/*! \brief Converts \c IfcSurface to a according to subtype.
 					\param		surface	\c IfcSurface entity to be interpreted.
 					\param		pos
-					\param		polylineData
+					\return		polylineData
 					\note		The \c IfcSurface subtypes are \c IfcBoundedSurface,  \c IfcElementarySurface and  \c IfcSweptSurface.
 				*/
 
-				carve::input::PolylineSetData convertIfcSurface(const EXPRESSReference<typename IfcEntityTypesT::IfcSurface>& surface,
+				std::shared_ptr<carve::input::PolylineSetData> convertIfcSurface(const EXPRESSReference<typename IfcEntityTypesT::IfcSurface>& surface,
 					const carve::math::Matrix& pos) const throw(...) 
 					{
 
 					// (1/3) IfcBoundedSurface SUBTYPE of IfcSurface
 					if (surface.isOfType<typename IfcEntityTypesT::IfcBoundedSurface>()) {
-						polylinedata = convertIfcBoundedSurface(surface.as<typename IfcEntityTypesT::IfcBoundedSurface(),
+						return std::shared_ptr<carve::input::PolylineSetData> polylineData = convertIfcBoundedSurface(surface.as<typename IfcEntityTypesT::IfcBoundedSurface(),

```

Comment:
```suggestion
						return convertIfcBoundedSurface(surface.as<typename IfcEntityTypesT::IfcBoundedSurface(),
```
If the function returns `std::shared_ptr<carve::input::PolylineSetData>`, there is no need for intermediate variable.

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/151/commits/938258b12cd11ad44dcff830ecf58d638179c2cd